packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -2425,9 +2429,13 @@ typedef PopInvokedCallback = void Function(bool didPop);
 ///  * [PopScope], which provides similar functionality in a widget.
 ///  * [ModalRoute.registerPopEntry], which unregisters instances of this.
 ///  * [ModalRoute.unregisterPopEntry], which unregisters instances of this.
-abstract class PopEntry {
-  /// {@macro flutter.widgets.PopScope.onPopInvoked}
-  PopInvokedCallback? get onPopInvoked;
+abstract class PopEntry<T> {
+  /// {@macro flutter.widgets.PopScope.onPopWithResultInvoked}
+  // This can't be a function getter since the Dart VM doesn't allow upcasting
```

Comment:
It looks like this doesn't have any other implications in this PR, does it? Like I seem to see PopEntry is still being used in the same way.